cartographer/mapping_2d/sparse_pose_graph/constraint_builder.cc

```diff
@@ -162,14 +162,14 @@ void ConstraintBuilder::ComputeConstraint(
     const mapping::TrajectoryNode::Data* const constant_data,
     const transform::Rigid2d& initial_relative_pose,
     std::unique_ptr<ConstraintBuilder::Constraint>* constraint) {
-  const transform::Rigid2d initial_pose =
+  const transform::Rigid2d local_pose =
```

Comment:
I wonder what a good naming is here. This an `pose_estimate` are in local map frame. One is the starting point, the other the result of the scan matchers.